src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Search/NewAzureOperationalInsightsComputerGroupCommand.cs

```diff
@@ -73,11 +73,6 @@ protected override void ProcessRecord()
                 Tags = new List<Tag>() { new Tag() { Name = "Group", Value = "Computer" } }
             };
 
-            if (!SearchCommandHelper.IsListOfComputers(this.Query))
```

Comment:
The verification is removed because of a recent new feature in our service backend. Previously, a computer group query must end with either "measure count() by Computer" or "distinct Computer". We recently added a new feature that allows customers to specify a computer simply by a list of comma-separated values, such as "C1, C2, C3". With the change, the original verification is no longer needed and thus removed in this change.